internal/controllers/clusterrequest_controller_test.go

```diff
@@ -443,7 +443,13 @@ func removeRequiredFieldFromClusterInstanceCm(
 func removeRequiredFieldFromClusterInstanceInput(
 	ctx context.Context, c client.Client, crName, crNamespace string) {
 	// Remove required field hostname
-	currentCR := &oranv1alpha1.ClusterRequest{}
+	currentCR := &oranv1alpha1.ClusterRequest{
+		Spec: oranv1alpha1.ClusterRequestSpec{
+			Timeout: oranv1alpha1.Timeout{
+				HardwareProvisioning: 1,
+			},
+		},
```

Comment:
I think this isn’t needed since it was configured previously. The queried ClusterRequest should already have this set

Reply:
Removed.